cpu/pmqa.py.data/pmqa.yaml

```diff
@@ -0,0 +1,7 @@
+#Usage: run_arg can  any of one from below list
```

Comment:
@apahim thanks , but in result get any command status but i want to grep Fail as well. that's  why checking fail log in file